### PR TITLE
[Bug] Update hostname to valid metric name

### PR DESCRIPTION
**Pull request by colinmjj**
### What changes were proposed in this pull request?
As metric name, host name has invalid character like `-`, `.`
Replace them with `_` to generate valid metric name

### Why are the changes needed?
Bug fix

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
UT added

### CODE REVIEW

File: common/src/main/java/com/tencent/rss/common/util/RssUtils.java [hunk 1]
```diff
@@ -267,4 +267,14 @@ public static void checkQuorumSetting(int replica, int replicaWrite, int replica
...
   }
+
+  // the method is used to transfer hostname to metric name.
+  // With standard of hostname, `-` and `.` will be included in hostname,
+  // but they are not valid for metric name, replace them with '_'
+  public static String getMetricNameForHostName(String hostName) {
+    if (hostName == null) {
+      return "";
+    }
+    return hostName.replaceAll("[\\.-]", "_");
```
Comment (frankliee) [inline common/src/main/java/com/tencent/rss/common/util/RssUtils.java:278]: What about ' '(i.e., blank)?

Comment (colinmjj) [reply common/src/main/java/com/tencent/rss/common/util/RssUtils.java:278]: For hostname, standards(eg, RFC 1035) which only contains `[a-z], -, .` are applied, the other character is not allowed. If the rule is broken, update the method according to new rule.

Comment (frankliee) [reply common/src/main/java/com/tencent/rss/common/util/RssUtils.java:278]: That is OK.
LT2M.

**Review by frankliee — APPROVED** [review]

**Comment by colinmjj** [comment]
What changes were proposed in this pull request?
As metric name, host name has invalid character like -, .
Replace them with _ to generate valid metric name

Why are the changes needed?
Bug fix

Does this PR introduce any user-facing change?
No

How was this patch tested?
UT added

**Pull request merged**